src/ducks/billsMatching/Linker/billsToOperation/helpers.js

```diff
@@ -0,0 +1,78 @@
+const sortBy = require('lodash/sortBy')
```

Comment:
Not necessary to review the whole file in this commit, it is basically a copy/paste from cozy-konnector-libs at this stage.